doc/api-extensions.md

```diff
@@ -420,10 +420,6 @@ files for the container via `nofile`. The format is `limits.kernel.[limit name]`
 
 This adds support for renaming custom storage volumes.
 
-## `external_authentication`
```

Comment:
Interesting that the name of the extension doesn't match shared/version/api.go, maybe send a fix to LXD to line them up so they don't hit an issue at some point?

Reply:
I've sent a PR.